### ISSUE TITLE
Network graph - display data issue (covered point)

### ISSUE THREAD

**Issue by raf18seb** [highcharts/highcharts#10194]
#### Expected behaviour
I expect all points visible with this set of data:
```
data: [
      ["0", "1"],
      ["0", "2"],
      ["0", "3"],
      ["0", "4"],
      ["0", "5"],
      ["0", "6"],
      ["0", "7"],
      ["0", "8"],
      ["0", "9"],
      ["0", "10"],
      ["0", "11"],
      ["0", "12"],
      ["0", "13"],

      ["14", "15"],
      ["14", "16"],
      ["17", "18"],
      ["17", "19"],
      ["20", "21"],
      ["20", "22"]
    ]
```

#### Actual behaviour
The **["17", "19"]** point is hidded (covered) by another point.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/32zotp0y
Note, that if you remove **["0", "13"]** point, the data is displayed correctly.

#### Product version
Highcharts 7.0.3

#### Affected browser(s)
IE, Edge, Chrome.

The behavior is  slightly differently on **Firefox**.

**Comment by pawelfus** [comment]
Will be fixed in v7.1:
- verlet: https://jsfiddle.net/BlackLabel/32zotp0y/13/
- https://jsfiddle.net/BlackLabel/32zotp0y/14/

Note: Issue is possible to reproduce on retina + fixed width: https://jsfiddle.net/BlackLabel/32zotp0y/16/

**Comment by pawelfus** [comment]
Closing since it's fixed in v7.1 branch.
